src/lib/booleans.js

```diff
@@ -0,0 +1,92 @@
+function negate(a) {
+  return !a;
+};
+
+function both(a, b) {
+  return a&&b;
```

Comment:
You should run a linter over this file, a lot of it has very condensed code that isnt that easy to read. This one for example shouold be:
```
  return a && b;
```

As with the last file, the commented out duplicates of work you already have should be deleted